ytsync/setup.go

```diff
@@ -30,31 +30,32 @@ func (s *Sync) walletSetup() error {
 	balance := decimal.Decimal(*balanceResp)
 	log.Debugf("Starting balance is %s", balance.String())
 
-	var numOnSource uint64
+	var numOnSource int
 	if s.LbryChannelName == "@UCBerkeley" {
 		numOnSource = 10104
```

Comment:
what is this?! not part of the PR but this is not good to have in the code base.

Reply:
I can explain the berkeley stuff to you via DM, it's all good as it will be eventually removed from here. Not worth changing now